spec/controllers/users_controller.spec.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :controller do
+  describe 'GET#index' do
+    it 'renders a successful response' do
+      get :index
```

Comment:
I think we should add a line break between the test commands and expectations

Reply:
Fixed on 0019211be7c8b967e4a9fac03b98ade2924ec014